src/styles/pages/demo.scss

```diff
@@ -15,30 +31,147 @@
         margin: 18px;
       }
     }
+    .cart-count {
+      width: 18px;
+      height: 18px;
+      border-radius: 50%;
+      margin-left: 4px;
+      background: #000000;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
+
+    .cart-count.text {
```

Comment:
Let's move this into the block above by writing 

```scss
    .cart-count {
      width: 18px;
      height: 18px;
      border-radius: 50%;
      margin-left: 4px;
      background: #000000;
      display: flex;
      align-items: center;
      justify-content: center;

      &.text {
        color: #fff;
        font-weight: 900;
        font-size: 14px;
        line-height: 16px;
      }
    }
```

This formulation makes it clearer that those blocks are related and would reduce the number of times you have to explicitly declare the `cart-count` class.

Reply:
Nice. made this change.